src/status_im/feature_flags.cljs

```diff
@@ -20,6 +20,7 @@
    ::wallet.assets-modal-hide           (enabled-in-env? :FLAG_ASSETS_MODAL_HIDE)
    ::wallet.assets-modal-manage-tokens  (enabled-in-env? :FLAG_ASSETS_MODAL_MANAGE_TOKENS)
    ::wallet.bridge-token                (enabled-in-env? :FLAG_BRIDGE_TOKEN_ENABLED)
+   ::wallet.buy-recurrent-assets        (enabled-in-env? :FLAG_BUY_RECURRENT_ASSETS)
```

Comment:
Is the feature flag specifically for this feature or for the entire swaps feature? I just wonder about the granularity of feature flags we want to have for the new features (swap/dapps).

Reply:
this is not related to swaps 👍 it is just a feature in the "core" wallet set

Reply:
I thought it was swaps cause both issues that this PR closes have the `swaps-team` label 🤷‍♂️